### DISCUSSION TITLE
Option to automatically enable moderator mode

### DISCUSSION THREAD

**Discussion by srnyx** [Chatterino/chatterino2#3557] [Ideas]
An option to automatically enable moderator mode when opening Chatterino.
This would fix #1819

**Comment by pajlada** [comment]
It's a roundabout way to solve the bug that is: Detached splits (e.g. from extensions) are recreated, causing them to lose states like filters/mod mode etc.

I'd rather have a proper solution than adding yet another setting

**Reply by srnyx** [reply]
Agreed. But this setting would still be useful even if you aren't using the extension.
